src/map/skill.c

```diff
@@ -4106,6 +4106,13 @@ static int skill_timerskill(int tid, unsigned int tick, int id, intptr_t data)
 						skill_attack(skl->type,src,src,target,skl->skill_id,skl->skill_lv,tick,skl->flag);
 						break;
 					}
+				case SC_ESCAPE:
+					if (skl->type < 4 + skl->skill_lv) {
```

Comment:
Wouldn't it be nicer to read that from the skill_db column you removed the entries from instead of hardcoding it?